cli/cmd/root.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "cft",
+	Short: "cft",
```

Comment:
As per [examples](https://github.com/spf13/cobra/blob/master/cobra/cmd/init.go#L29), short should be a short description (probably line 14 works) while `Long` is a full explanation of functionality.

Reply:
updated